pkg/service/user.go

```diff
@@ -972,7 +971,11 @@ func (s *userService) RetrieveCliConfig(ctx context.Context, req *userrpcv3.ApiK
 	// get the default project associated to this account
 	ap, err := dao.GetDefaultAccountProject(ctx, s.db, uuid.MustParse(req.Id))
 	if err != nil {
-		return nil, err
+		_log.Debug("unable to fetch account permissions for default project, looking further..")
```

Comment:
It should be `info` log.

Reply:
debatable, but doesnt harm to mark it Info. going ahead.